Seminars/Homeworks/Homework_002/Task_005.py

```diff
@@ -0,0 +1,20 @@
+# Реализуйте алгоритм перемешивания списка.
+
+from random import randint
+
+def list(n):
+    list = []
+    for i in range(n):
+        list.append(randint(-n, n))
+    return list
+
+n = int(input('Введите число чтобы указать число элементов: '))
+numbers = list(n)
+print(numbers)
+for i in range (len(numbers)):
+    random_num = randint(i, n-1)
+    temp = numbers[i]
+    numbers[i] = numbers[random_num]
+    numbers[random_num] = temp
```

Comment:
```suggestion
    numbers[i], numbers[random_num] = numbers[random_num], numbers[i]
```
Это же Python, пользуйтесь его преимуществами над другими языками. Такого вы больше нигде не встретите.